OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -63,13 +61,13 @@ public CellTrigger(int id, CPos[] footprint, Action<Actor> onActorEntered, Actio
 				Dirty = true;
 			}
 
-			public void Tick(ActorMap am)
+			public void Tick(ActorMap actorMap)
 			{
 				if (!Dirty)
 					return;
 
-				var oldActors = currentActors;
-				currentActors = Footprint.SelectMany(c => am.GetUnitsAt(c)).ToList();
+				var oldActors = currentActors.ToList();
```

Comment:
This forces a(n unnecessary) copy of the list.  More than just a simple touchup!